bp/parser.js

```diff
@@ -42,10 +45,6 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
     result.rtype = 'RECORD';
     result.mime = 'HTML';
     result.unitid = param.productid;
-  } else if (/^\/$/i.test(path)) {
```

Comment:
Don't forget to delete this in analogist and in the test file if you don't want to detect it.